app/controllers/requests_controller.rb

```diff
@@ -31,7 +31,11 @@ def notify_users(message)
 
   def successfully_saved(format, request)
     notify_users("New VM request:\n" + request.description_text)
-    format.html { redirect_to @request, notice: 'Request was successfully created.' }
+    if current_user.role == 'admin'
+      format.html { redirect_to @request, notice: 'Request was successfully created.' }
+    else
+      format.html { redirect_to requests_url, notice: 'Request was successfully created.' }
```

Comment:
According to #155 the user should be redirected to the dashboard, not the requests overview